app/src/main/java/com/globant/example/mentorapp/home/presentation/view/adapter/ListUsersAdapter.java

```diff
@@ -50,16 +53,28 @@ public int getItemCount() {
         return users != null ? users.size() : context.getResources().getInteger(R.integer.zero_constant);
     }
 
-    public static class ViewHolder extends RecyclerView.ViewHolder {
+    static class ViewHolder extends RecyclerView.ViewHolder {
 
         private TextView userNameTextView;
         private ImageView userImage;
 
-        public ViewHolder(View itemView) {
+        public ViewHolder(View itemView, final Context context) {
             super(itemView);
             userNameTextView = (TextView) itemView.findViewById(R.id.textName);
             userImage = (ImageView) itemView.findViewById(R.id.imageUser);
+            itemView.setOnClickListener(new View.OnClickListener() {
+                @Override
+                public void onClick(View v) {
+                    UserDetailsFragment detailsFragment = UserDetailsFragment.getInstance();
```

Comment:
Avoid doing this. Click events in adapters MUST be managed in the Activity or Fragment, the adapter should never manage the navigation of the app.